board/app/controllers/comment_controller.php

```diff
@@ -0,0 +1,73 @@
+<?php
+class CommentController extends AppController {
+
+const MAX_COMMENT_PER_PAGE = 5;
+
+    public function write() 
+    {
+        $thread_id = Param::get('thread_id');
+
+        $thread = Thread::get($thread_id);
+        $comment = new Comment();
+        $current_page = Param::get('page_next', 'write');
+
+        switch($current_page) { 
+            case 'write':
+            break;
+
+        case 'write_end':                
+            $comment->body = Param::get('body');
+            $comment->username = Param::get('username');
+                
+            try {            
+                $comment->write($comment, $thread_id);
```

Comment:
you may remove $comment as parameter in this method since they're the same instance.